README.md

```diff
@@ -92,8 +92,26 @@ $ shuttle run build tag=v1
 * ...
 
 ## Documentation
-*Documentation is coming*
+*Documentation is under development*
 
+### Git Plan
+When using a git plan a url should look like:
+
+* `https://github.com/lunarway/shuttle-example-go-plan.git`
+* `git@github.com:lunarway/shuttle-example-go-plan.git`
+* `https://github.com/lunarway/shuttle-example-go-plan.git#change-build`
+* `git@github.com:lunarway/shuttle-example-go-plan.git#change-build`
+
+The `#change-build` points the plan to a specific branch, which by default would be `master`.
+It can also be used to point to a tag or a git SHA.
```

Comment:
Could you add an example of the Git SHA and tags as well?

Judging by the next section I expect it to be like `shuttle-example-go-plan.git#my-tag`? If this is the case, what happens if a tag and branch name conflicts? E.g. a `next` branch with a `next` tag?

Reply:
Not totally sure what will happen, but it will defaults to git's behavior. I think that means that branches has precedence over the tags.

Reply:
All right. It might be an unlikely scenario, so let's just leave it as is :)